code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm

```diff
@@ -120,6 +121,527 @@ datum/reagent/toxin/slimejelly/on_mob_life(var/mob/living/M as mob)
+		holder.remove_reagent(src.id,15)
+datum/reagent/toxin/volt
+	name = "Sparking mixture"
+	id = "volt"
+	description = " A bubbling concoction of sparks and static electricity."
+	color = "#11BFAC"
+datum/reagent/toxin/volt/on_mob_life(var/mob/living/M as mob)
+	if(prob(20))
+		for(var/mob/living/T in view(M.loc,6))
+			var/obj/effect/lightning/eff = new /obj/effect/lightning(T.loc)
+
+			eff.start()
+			T.adjustFireLoss(15)
+			T.adjustBrainLoss(10)
+		playsound(M.loc,'sound/effects/thunder.ogg',50,1)
+		holder.remove_reagent(src.id, 10, safety = 1)
+datum/reagent/toxin/volt/on_fart(var/mob/living/M as mob)
+	var/obj/effect/lightning/eff = new /obj/effect/lightning(M.loc)
+	eff.start()
+	playsound(M.loc,'sound/effects/thunder.ogg',50,1)
+	M.gib()
+datum/reagent/toxin/emit_on
+	name = "Glowing Emittrium"
+	id = "emit_on"
+	description = "Run!"
+	color = "#1211FB"
+datum/reagent/toxin/emit_on/on_update()
+	var/emit_dir = pick(1,2,3,4)
+	var/obj/item/projectile/beam/emitter/A = PoolOrNew(/obj/item/projectile/beam/emitter,get_turf(holder.my_atom))
+	switch(emit_dir)
+		if(1)
+			A.yo = 20
+			A.xo = 0
+		if(2)
+			A.yo = 0
+			A.xo = 20
+		if(3)
+			A.yo = 0
+			A.xo = -20
+		else
+			A.yo = -20
+			A.xo = 0
+	A.fire()
+	holder.remove_reagent(src.id,10,safety = 1)
+datum/reagent/toxin/emit_on/on_mob_life(var/mob/living/M as mob)
+	M.adjustFireLoss(10)
+	var/emit_dir = pick(1,2,3,4)
+	var/obj/item/projectile/beam/emitter/A = PoolOrNew(/obj/item/projectile/beam/emitter,get_turf(holder.my_atom))
+	switch(emit_dir)
+		if(1)
+			A.yo = 20
+			A.xo = 0
+		if(2)
+			A.yo = 0
+			A.xo = 20
+		if(3)
+			A.yo = 0
+			A.xo = -20
+		else
+			A.yo = -20
+			A.xo = 0
+	A.fire()
+	holder.remove_reagent(src.id,10,safety = 1)
+datum/reagent/toxin/emit_on/on_fart(var/mob/living/M as mob)
+	M.adjustFireLoss(50)
+	var/obj/item/projectile/beam/emitter/A = PoolOrNew(/obj/item/projectile/beam/emitter,get_turf(holder.my_atom))
+	var/obj/item/projectile/beam/emitter/B = PoolOrNew(/obj/item/projectile/beam/emitter,get_turf(holder.my_atom))
+	var/obj/item/projectile/beam/emitter/C = PoolOrNew(/obj/item/projectile/beam/emitter,get_turf(holder.my_atom))
+	var/obj/item/projectile/beam/emitter/D = PoolOrNew(/obj/item/projectile/beam/emitter,get_turf(holder.my_atom))
+	A.yo = 20
+	A.xo = 0
+	B.yo = 0
+	B.xo = 20
+	C.yo = 0
+	C.xo = -20
+	D.yo = -20
+	D.xo = 0
+	A.fire()
+	B.fire()
+	C.fire()
+	D.fire()
+	holder.remove_reagent(src.id,100,safety = 1)
+datum/reagent/toxin/cryonium
+	name = "Cryonium"
+	id = "cryo"
+	description = "Chilly."
+	color = "#75AC53"
+datum/reagent/toxin/cryonium/on_mob_life(var/mob/living/M as mob)
+	M.bodytemperature -= 3//this is how you avoid super boom
+datum/reagent/toxin/cryonium/on_update()
+	if(!holder)
+		return
+	if(holder.present_machines[1]-5 >= 0)
+		holder.present_machines[1] -= 5
+	else
+		holder.present_machines[1] -= (5- holder.present_machines[1])
+	holder.remove_reagent(src.id,5)
+datum/reagent/toxin/pyrosium
+	name = "Pyrosium"
```

Comment:
if you're not gonna make it work like pyrosium don't name it pyrosium

it's just going to confuse the hell out of people coming in from other servers/leaving for other servers